TodoListWebApp/Global.asax.cs

```diff
@@ -17,9 +17,6 @@ protected void Application_Start()
             FilterConfig.RegisterGlobalFilters(GlobalFilters.Filters);
             RouteConfig.RegisterRoutes(RouteTable.Routes);
             BundleConfig.RegisterBundles(BundleTable.Bundles);
-            
-            //when deployed to Azure, modifying the security protocol seems necessary now:
-            System.Net.ServicePointManager.SecurityProtocol = System.Net.SecurityProtocolType.Tls12;
```

Comment:
This is the only code change, right? i.e. to leave crossing the right TLS version to the .NET runtime ?